### DISCUSSION TITLE
bt下载活动链接4000左右，导致实时信息中链接无法查看

### DISCUSSION THREAD

**Discussion by liuxichi** [immortalwrt/immortalwrt#852] [General]
/usr/lib/lua/luci/dispatcher.lua:427: /etc/config/luci seems to be corrupt, unable to find section 'main'会出现这样的提示，请问怎么修复

**Comment by 1715173329** [comment]
如果刷新无效，重启即可。

**Comment by liuxichi** [comment]
> 如果刷新无效，重启即可。

谢谢大佬回复，我意思是什么原因导致的，能用什么方式修改或者优化吗？这也算是一个小bug

**Reply by 1715173329** [reply]
![image](https://user-images.githubusercontent.com/22235437/213167787-bc998d0c-f1bc-4c0b-83f2-8db899cedcab.png)
祖传 bug。

**Reply by liuxichi** [reply]
谢谢大佬！ nlbw也是bt链接一多，不管设置多大的rmem_max一律提示内存不足。如果能限制记录最近1万条链接就好了，可能会有帮助吧！
